resources/metadata/abapEnvironmentAssemblePackages.yaml

```diff
@@ -0,0 +1,129 @@
+metadata:
+    name: abapEnvironmentAssemblePackages
+    description: Assembly of installation, support package or patch in SAP Cloud Platform ABAP Environment system
+    longDescription: |
+      This step runs the assembly of a list of provided installations, support packages or patches in SAP Cloud
+      Platform ABAP Environment system and saves the corresponding SAR archive to the filesystem.
+spec:
+  inputs:
+    secrets:
+      - name: abapCredentialsId
```

Comment:
`username` and `password` should be linked to this, see https://github.com/SAP/jenkins-library/pull/1961